idea_town/frontend/static-src/app/views/landing_page.js

```diff
@@ -0,0 +1,37 @@
+import app from 'ampersand-app';
+import AmpersandView from 'ampersand-view';
+import mustache from 'mustache';
+
+// TODO replace with an api endpoint that exposes idea town addon info
+const addonInfo = {
+  name: 'Idea Town',
+  url: '//localhost/codez/github/mozilla-idea-town-addon/@idea-town-addon-0.0.1.xpi'
```

Comment:
> Until we have this provided by an API, I'll move the actual URL to settings.py, and have the server inject the path along with the other unmentionables we're currently injecting into the page itself.

I wound up just inlining the path in the template itself. Using settings.py seemed like a not-useful level of indirection for a very temporary hack.